### PR TITLE
Make fs_inotify_max_user_watches configurable.

### PR DESCRIPTION
**Pull request by aveshagarwal**
@sjenning

### CODE REVIEW

**Comment by jmencak** [comment]
Thanks for the PR!  So, you're changing the "fs.inotify.max_user_watches" only on the worker nodes (not the control plane).  Is this intentional?

**Comment by aveshagarwal** [comment]
> Thanks for the PR! So, you're changing the "fs.inotify.max_user_watches" only on the worker nodes (not the control plane). Is this intentional?

Yes.

**Comment by sdodson** [comment]
/approve
lgtm too but didn't know if anyone else wanted to review before it merges

File: roles/tuned/defaults/main.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ---
 tuned_etc_directory: '/etc/tuned'
 tuned_templates_source: '../templates'
+openshift_fs_inotify_max_user_watches: 65536
```
Comment (wmengRH) [inline roles/tuned/defaults/main.yml]: @sjenning  65536 here is different from 
fs.inotify.max_user_watches = 524288 in comment
https://bugzilla.redhat.com/show_bug.cgi?id=1559061#c3
I am confused.
Is this intended? 
Thanks.

Comment (aveshagarwal) [reply roles/tuned/defaults/main.yml]: I just put 65536 as a default value as in my understanding 65536 is the default value of  fs.inotify_max_user_watches. If you want to change default value and have valid reasons, I am fine with it.

Comment (wmengRH) [reply roles/tuned/defaults/main.yml]: got it. Thanks for explanation.

**Pull request merged**